tests/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "codebuild_id" {
+  default = ""
+}
+
+variable "KeyPairName" {
```

Comment:
prefer the pythonic style variables names, .e.g `keypair_name`

Reply:
updated